src/bundle/Resources/views/themes/admin/content/edit_macros.html.twig

```diff
@@ -0,0 +1,23 @@
+{% macro renderFormField(formField) -%}
+    {% if not formField.rendered %}
+        {% if formField.value is defined %}
+            {{- form_widget(formField) -}}
+        {% else %}
+            <div class="ibexa-field-edit ibexa-field-edit--eznoneditable">
+                {{- form_label(formField) -}}
+                <p class="non-editable">
+                    {{ 'content.field.non_editable'|trans|desc('This Field Type is not editable') }}
+                </p>
+                {% do formField.setRendered() %}
+            </div>
+        {% endif %}
+    {% endif %}
+{%- endmacro -%}
+
+{% macro renderFieldGroup(group, form) -%}
+    {% for field in group %}
+        {% set formField = form.fieldsData[field] %}
+
+        {{ _self.renderFormField(formField) }}
+    {% endfor %}
+{%- endmacro -%}
```

Comment:
To be fair, the "correct" way of doing custom form field rendering is by using blocks.

Reply:
How? Given it is inside _embed_.

Reply:
Ok, maybe with symfony form blocks, but we do not have it here now afaics.